### PR TITLE
Workingbranch st2180

### PR DESCRIPTION
**Pull request by st2180**
Implement a test for _As a user I want to be able to view my accounts so that I can see my balances_.

### CODE REVIEW

**Review by sjt-85 — CHANGES_REQUESTED** [review]
Hi Shinichi,

Looking good.  I'll have a go at making my own test classes later using your new feature.

Just a couple of things to tweak:
1.  can you fix the indentation to 2 spaces please?
2.  it's generating warnings in Eclipse due to the use of the raw type Class ("Class is a raw type.  References to generic type Class\<T\> should be parameterized").  It seems happy with Class\<?\> instead of Class.  Could you have a look please and check it all works with Class\<?\>?

Thanks

**Comment by st2180** [comment]
I format the code using a tool, and I change the type Class to Class<?>. Could you check them out?

File: newbank/test/ServerTestScenarios.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// 3. register the test class the runInTestMode method.
+public class ServerTestScenarios {
+
+  public void ShowMyAccountsReturnsListOfAllCustomersAccountsAlongWithCurrentBalance() {
+
+    String bhagy =
+        NewBank.getBank()
+            .processRequest(NewBank.getBank().checkLogInDetails("Bhagy", null), "SHOWMYACCOUNTS");
+
+    newbank.test.NBUnit.AssertEqual("Main: 1000.0" + System.lineSeparator(), bhagy);
```
Comment (sjt-85) [inline newbank/test/ServerTestScenarios.java:20]: Hi Shinichi sorry one last thing then it's ready to merge... to make your test compatible with the password feature recently added to sprint1 can you change the test login credentials to checkLogInDetails("Bhagy", **"1"**) and checkLogInDetails("Christina", **"2"**) please?

**Comment by st2180** [comment]
Hi Simon, I changed the things and a few others. Thanks.

**Review by sjt-85 — APPROVED** [review]

**Pull request merged**